core/src/main/java/jenkins/util/ResourceBundleUtil.java

```diff
@@ -72,14 +74,39 @@ private ResourceBundleUtil() {
             return bundleJSON;
         }
 
-        ResourceBundle bundle = ResourceBundle.getBundle(baseName, locale);
+        ResourceBundle bundle = getBundle(baseName, locale, Jenkins.class.getClassLoader());
+        if (bundle == null) {
+            // Not in Jenkins core. Check the plugins.
+            Jenkins jenkins = Jenkins.getInstance();
+            if (jenkins != null) {
```

Comment:
Actually it is @oleg-nenashev the failure you have linked is provoked by that test not being in place anymore.